source/common/filter/config_discovery_impl.h

```diff
@@ -87,7 +87,8 @@ class DynamicFilterConfigProviderImpl : public DynamicFilterConfigProviderImplBa
     if (!tls->isShutdown()) {
       tls->runOnAllThreads([](OptRef<ThreadLocalConfig> tls) { tls->config_ = {}; },
                            // Extend the lifetime of TLS by capturing main_config_, because
-                           // otherwise, the callback to clear TLS worker content is not executed.
+                           // otherwise, the callback to clear TLS worker content is not
```

Comment:
Please revert formatting change that are not relevant.

Reply:
When I run local formatter - it changed it like this. I can revert it though.